helpers/lib_tasks_docker_release.py

```diff
@@ -611,18 +620,68 @@ def docker_build_prod_image(  # type: ignore
     opts = "--no-cache" if not cache else ""
     # Use dev version for building prod image.
     dev_version = hlitadoc.to_dev_version(prod_version)
-    cmd = rf"""
-    DOCKER_BUILDKIT={DOCKER_BUILDKIT} \
-    time \
-    docker build \
-        {opts} \
-        --tag {image_versioned_prod} \
-        --file {dockerfile} \
-        --build-arg VERSION={dev_version} \
-        --build-arg ECR_BASE_PATH={os.environ["CSFY_ECR_BASE_PATH"]} \
-        .
-    """
-    hlitauti.run(ctx, cmd)
+    # Build for both a single arch or multi-arch.
+    # TODO(Vlad): Refactor with the `docker_build_local_image()`.
```

Comment:
Ok, but let's try to do the refactoring as part of the fix, otherwise the tech debt just increases.

Can we refactor this? If not now, we should do immediately after this PR.